masterlist.yaml

```
@@ -9442,13 +9442,27 @@ plugins:
             text: 'Consider using Creature Diversity.esp or MMM or Frans instead.'
           - lang: de
             text: 'Erwägen Sie stattdessen Creature Diversity.esp oder MMM oder Frans zu nutzen.'
+
   - name: 'WCGL - Wilderness Creatures 1.1.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/1969' ]
     inc:
       - 'FCOM_Francescos.esp'
       - 'Francesco''s Leveled Creatures-Items Mod.esm'
       - 'Mart''s Monster Mod.esm'
       - 'Oscuro''s_Oblivion_Overhaul.esp'
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Delete the unused record **01000CE7** using TES4Edit, blocks merging into the Bashed Patch.' ]
+        condition: 'checksum("WCGL - Wilderness Creatures 1.1.esp", 187692FD)'
```

@Infernio Actually, since this is something that only affects people using a Bashed Patch & is harmless otherwise, I think it'd be best to only display it if they're using one.

```suggestion
        condition: 'active("Bashed Patch.*\.esp") and checksum("WCGL - Wilderness Creatures 1.1.esp", 187692FD)'
```

---

Agreed 👍